app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -123,6 +123,8 @@ def process_vm_instance(object)
     vm_object = process_domain(object.namespace, object.memory, object.cpu_cores, uid, name)
     process_status(vm_object, object.ip_address, object.node_name)
 
+    vm_object.host_id = ManageIQ::Providers::Kubevirt::InfraManager::Host.find_by(:name => object.node_name).id
+
```

Comment:
We need to relate the vms to the host they are on for `#capture_ems_targets`
https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/260/files#diff-b1f5b6941dfd5a33805573358fffd6341b7cd68d3806dce75674dca8827f60e2R49

In this case it would be the node on which the vm is deployed on

Reply:
This probably won't work on the very first refresh since I'm querying the model, @agrare is there a better way to go about this?

Reply:
That's right typically we would `vm_object.host = persister.hosts.lazy_find(host_ems_ref)`
If you can't get the host ems_ref (which looks like the uid) there are options but using the ems_ref is ideal